src/transformers/modeling_roberta.py

```diff
@@ -184,10 +184,9 @@ def forward(
         position_ids=None,
         head_mask=None,
         inputs_embeds=None,
-        masked_lm_labels=None,
```

Comment:
I think this was deleted accidently

Reply:
No, it was merely renamed to `labels`, so this change seems to be correct. Unfortunately this is showing up separately after I fixed conflicts in the merge commit

Reply:
gottcha! Never mind my comment then :-)